include/openssl/cmp.h.in

```diff
@@ -329,8 +329,8 @@ int OSSL_CMP_CTX_build_cert_chain(OSSL_CMP_CTX *ctx, X509_STORE *own_trusted,
 int OSSL_CMP_CTX_set1_pkey(OSSL_CMP_CTX *ctx, EVP_PKEY *pkey);
 int OSSL_CMP_CTX_set1_referenceValue(OSSL_CMP_CTX *ctx,
                                      const unsigned char *ref, int len);
-int OSSL_CMP_CTX_set1_secretValue(OSSL_CMP_CTX *ctx, const unsigned char *sec,
-                                  const int len);
+int OSSL_CMP_CTX_set1_secretValue(OSSL_CMP_CTX *ctx,
+                                  const unsigned char *sec, int len);
```

Comment:
It's a bit unfortunate that your whitespace reformatting hides the aktual fix, namely that the `const` was removed.

I suggest you create two commits (in this order, to facilitate `git blame`):

-  one commit for the typo and the whitespace changes
-  one commit for fixing the compiler-warning